modin/pandas/test/dataframe/test_reduce.py

```diff
@@ -181,6 +181,14 @@ def test_2195(datetime_is_numeric, has_numeric_column):
     )
 
 
+def test_describe_different_index():
+    modin_df = pd.DataFrame(np.zeros((2, 60)))
+    pandas_df = pandas.DataFrame(np.zeros((2, 60)))
+    modin_df["new"] = pandas_df["new"] = "abc"
+    df_equals(modin_df.describe(include="all"), pandas_df.describe(include="all"))
+    df_equals(modin_df.describe(include="all").T, pandas_df.describe(include="all").T)
```

Comment:
```suggestion
```

I know the original bug report had the transpose, but I don't think checking the transpose gives us any more coverage.